game-core/src/main/java/games/strategy/net/MacFinder.java

```diff
@@ -216,7 +217,7 @@ private static String convertMacBytesToString(final byte[] mac) {
       return null;
     }
     return Joiner.on('.')
-        .join(FluentIterable.from(Bytes.asList(mac)).transform(macbyte -> String.format("%02X", macbyte)));
+        .join(Bytes.asList(mac).stream().map(macbyte -> String.format("%02X", macbyte)).collect(Collectors.toList()));
```

Comment:
```java
return Arrays.stream(mac)
  .map(macbyte -> String.format("%02X", macbyte))
  .collect(Collectors.joining("."));
```

Reply:
Hmm, while i agree with formatting update, the previous code was not formatted any different. I think we should reserve 'request changes' for when things are wrong or made worse, if none better than should be just a comment.

Reply:
Fixed in: https://github.com/triplea-game/triplea/pull/3150/commits/3ffc2c62c6de19e73d9beccd3f6a467146046855

Reply:
I'm Confused 😕 
The change I suggested was to use the streaming API exclusively instead of a weird mixture of it and guava.
Back when I implemented this I didn't know I could use the streaming API for this.

Reply:
Oh, please be more explicit, I did not realize that was more than a formatting suggestion.

Reply:
Sorry, I thought it was obvious enough I don't submit code snippets when it just comes to formatting 😅
Will do the next time 👍

Reply:
private static String convertMacBytesToString(final byte[] mac) {
    return Arrays.stream(mac)
        .map(macbyte -> String.format("%02X", macbyte))
        .collect(Collectors.joining("."));

^^ this does not work; get a compilation error as you cannot stream on a `byte[]`

Reply:
Wow, `Arrays.stream` has overloads for int arrays double arrays and long arrays, floats shorts and bytes are not being considered. 
That being said what should work is using `Bytes.asList(mac).stream()` instead of the arrays class